packages/discord-utilities/src/lib/limits.ts

```diff
@@ -95,6 +95,26 @@ export const GuildMemberLimits = {
 	MaximumDisplayNameLength: 32
 } as const;
 
+/**
+ * Namespace containing limits related to Discord interactions.
+ */
+export const InteractionLimits = {
+	/**
+	 * Maximum buttons allowed in a single action row.
+	 */
+	MaximumButtonsPerActionRow: 5,
+
+	/**
+	 * Maximum select menus allowed in a single action row.
+	 */
+	MaximumSelectMenusPerActionRow: 1,
+
+	/**
+	 * Maximum items allowed in a single select menu.
```

Comment:
Maximum options maybe? 🤔

Reply:
You mean max options in a slash command? I didn't know there was a limit, what is it please?

Reply:
Maximum Items in the JSDoc comment can be replaced with Maximum options, also the maximum number of options a slash command can have is 25, covering all types of options

Reply:
Oh sorry you meant in the JSDoc ! thanks for the suggestion